types/wordpress__blocks/api/registration.d.ts

```diff
@@ -167,3 +176,29 @@ export function unregisterBlockStyle(blockName: string, styleVariationName: stri
  * @returns The previous block value if successfully unregistered, otherwise `undefined`.
  */
 export function unregisterBlockType(name: string): Block<any> | undefined;
+
+/**
+ * Returns an array with the variations of a given block type.
+ *
+ * @param blockName - Name of block (example: “core/columns”).
+ * @param [scope] - Block variation scope name.
+ *
+ * @returns Block variations.
+ */
+export function getBlockVariations(blockName: string, scope?: BlockVariationScope): BlockVariation[] | undefined;
+
+/**
+ * Registers one or more new block variations for the given block type.
+ *
+ * @param blockName - Name of the block (example: “core/columns”).
+ * @param variation - Object or array of objects describing block variations.
+ */
+export function registerBlockVariation(blockName: string, variation: BlockVariation | BlockVariation[]): void;
+
+/**
+ * Unregisters one or more variations defined for the given block type.
+ *
+ * @param blockName - Name of the block (example: “core/columns”).
+ * @param variationName - Name or array of variation names defined for the block.
```

Comment:
Nitpick on wording again: "names defined for the block" reads to me like there could be multiple names per variation.

Would something like this clarify?

```ts
 * @param variationName - Name(s) of variations(s) to unregister.
```

Reply:
I've followed the first suggestion, I liked that pattern more. And appreciating these small nitpicks - English is not my first language and as much as I'm trying to flex my linguistics I've learned it long ago that whatever sounds good in my head might be far from being clear for the next person to lay their eyes on it, and that includes me the day after... 

Appreciated!